GMG/GMG/UI/Components/Metronome.swift

```diff
@@ -6,13 +6,269 @@
 //
 
 import SwiftUI
+import AVFoundation
+import Combine
 
 struct Metronome: View {
+    let bpm: BPM //bpm
+    let isPlaying: Bool //메트로놈 플레이 여부
+    let timeSignature: TimeSignature //박자
```

Comment:
```suggestion
    let bpm: BPM //bpm
    let isPlaying: Bool //메트로놈 플레이 여부
    let timeSignature: TimeSignature //박자
```

ㅋㅋㅋㅋ 위 내용들은 주석처리 안해도 잘 이해돼서 빼도 될듯해요!

Reply:
넹 바꿨어영